Resources/Prototypes/Entities/Objects/Weapons/Melee/knife.yml

```diff
@@ -88,3 +88,18 @@
     size: 10
     sprite: Objects/Weapons/Melee/combat_knife.rsi
     prefix: inhand
+
+- type: entity
+  name: survival knife
+  parent: CombatKnife
+  id: SurvivalKnife
+  description: Weapon of first and last resort for combatting space carp.
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Melee/survival_knife.rsi
+    size: 2
+    state: icon
+  - type: Item
+    size: 10
+    sprite: Objects/Weapons/Melee/survival_knife.rsi
+    prefix: inhand
```

Comment:
this should be weaker than the combat knife. maybe 13 damage, or buff the combat knife

Reply:
It currently kills a carp in 3 hits which seems fair? In that case I could buff combat knife to 20.

Reply:
Pretty big buff killing people from 7 hits into 5.

Reply:
I'm pretty sure the combat knife is an admin only spawn. I think the survival and combat knife can just stay as is, they both do the same amount of damage as the chef's clever anyway.

Reply:
Combat knife will probably be a vendor item for antags like ops in future so buffing is probs not a great idea. 7/8 hits on non armoured target seems fine? Damages will probably have to be rebalanced after body system anyway.

Reply:
Yep, I'll leave this as is then.